content/en/docs/reference/access-authn-authz/node.md

```diff
@@ -27,6 +27,11 @@ Read operations:
 * secrets, configmaps, persistent volume claims and persistent volumes related
   to pods bound to the kubelet's node
 
+{{< feature-state feature_gate_name="AuthorizeNodeWithSelectors" >}}
+
+When the `AuthorizeNodeWithSelectors` feature is enabled, kubelets are only allowed to
```

Comment:
is it just this gate or both gates required?

Reply:
startup validation won't _let_ you enable AuthorizeNodeWithSelectors without also enabling AuthorizeWithSelectors, tried to keep this doc focused on the specific gate enabling this capability

Reply:
+1 for adding a short note about enabling the other feature gate is needed to save users time to figure out why the feature isn't working

Reply:
done